### DISCUSSION TITLE
Ability to choose monthly as iteration duration

### DISCUSSION THREAD

**Discussion by abaiju15** [community/community#14631] [Projects and Issues]
Currently we are only able to pick `days`/`weeks` for the duration

<img width="446" alt="image" src="https://user-images.githubusercontent.com/9519851/162582841-abaca6a9-f802-4f4e-a947-19fd167032ad.png">

It would be nice to have a `months` option as well natively available so users don't have to customize each iteration duration to create them.

**Comment by samcheng24** [comment]
This would be good for teams that run on monthly/quarterly iterations

**Comment by mustafaozhan** [comment]
Agree would love to see that since we use monthly sprints, and it will help Quarterly roadmaps

**Comment by qqpann** [comment]
Want this!

**Comment by samox73** [comment]
Is there any update on this? It would be great to have!

**Comment by Pusik** [comment]
Please. Monthly. Variable duration better.
